webpack.config.js

```diff
@@ -12,7 +12,7 @@ const commonConfig = {
     module: {
         rules: [
             {
-                exclude: /(node_modules)/,
+                exclude: /node_modules(?!\/js-utils)/,
```

Comment:
This tells Webpack to not exclude js-utils and use babel-loader.

Reply:
Why do we need the babel loader for it?

Reply:
`js-utils` needs to use babel-loader. 
I got errors so I checked how `lib-jitsi-meet` uses it. I found something similar. https://github.com/jitsi/lib-jitsi-meet/blob/7b39059fba68240ac930af30cc99b655a3ba7edf/webpack.config.js#L46